deploy/operator.yaml

```diff
@@ -55,6 +55,11 @@ spec:
                 configMapKeyRef:
                   name: nmstate-config
                   key: interfaces_filter
+            - name: CONNECTIVITY_CHECK_ENDPOINT
```

Comment:
If I was a user I would have no idea what the endpoint means. Maybe connectivity_check_ip_target?

Reply:
But since it can be domain name, maybe just connectivity_check_target?

Reply:
ack, is good name too.